### ISSUE TITLE
Use associated constants in traits

### ISSUE THREAD

**Issue by bluss** [petgraph/petgraph#162]

**Comment by RaoulLuque** [comment]
Maybe someone else understands what this Issue is trying to address, but I am not sure what the associated constants are supposed to be ? Thus I'll nominate it as to be closed, but feel free to un-nominate it as describe it further if you disagree.

**Comment by starovoid** [comment]
I agree with the proposal to close this issue due to loss of relevance. It's been a long time, and there are also work on a trait's re-design: #559.
